pkg/operator/deploy/deploy.go

```diff
@@ -52,18 +54,28 @@ type CRD struct {
 // platform to deploy on, whether or not to delete the metering CRDs,
 // or namespace during an install, the location to the manifests dir, etc.
 type Config struct {
-	Namespace                string
-	Platform                 string
-	DeployManifestsDirectory string
-	MeteringCR               string
-	SkipMeteringDeployment   bool
-	DeleteCRDs               bool
-	DeleteCRB                bool
-	DeleteNamespace          bool
-	DeletePVCs               bool
-	DeleteAll                bool
-	Repo                     string
-	Tag                      string
+	SkipMeteringDeployment bool
+	DeleteCRDs             bool
+	DeleteCRB              bool
+	DeleteNamespace        bool
+	DeletePVCs             bool
+	DeleteAll              bool
+	Namespace              string
+	Platform               string
+	Repo                   string
+	Tag                    string
+}
+
+// MeteringResources contains all the objects that metering manages
+type MeteringResources struct {
+	CRDs               []CRD
+	Deployment         appsv1.Deployment
```

Comment:
Any reason we're not using pointers for all of these when we need to be passing them as references to all the functions that use them?